### DISCUSSION TITLE
Regarding Errors in packetArrived()

### DISCUSSION THREAD

**Discussion by ytrewq271828** [ANLAB-KAIST/KENSv3#154] [Project Q&A 2. Transport Layer (TCP)]
Hello, I have a question regarding errors that might happen in packetArrived().
We may encounter an error while reading the sender's IP address in IP layer packet, such as the return value of readData() not being matched with length parameter.
In those cases, are there any specific error handling scheme that we should follow? Or just throwing runtime error would be fine?

Thanks in advance.

**Comment by Pusnow** [comment]
If you receive malformed packets, you can just drop the packets. (You will implement this feature in PA3)

However, in PA2, there would not be malformed packets, so you probably parsed packets incorrectly.
